Updates settings for prototype 6

WalterKolczynski-NOAA opened the pull request:
Sample namelist: https://gist.github.com/WalterKolczynski-NOAA/7aabc8908edd50f1d3d0dbe2980438a6

Closes #277

WalterKolczynski-NOAA committed:
Update settings for prototype 6

Coupled cases are updated to the settings to be used for prototype 6.

For some settings, hard-coded values in parsing_namelists_FV3 had to
be replaced with existing environment variables.

Also cleaned up the fcst config yaml by removing redundant definitions
and renaming the variable name for the level settings.

Refs: #277

WalterKolczynski-NOAA committed:
Update model tag for prototype 6

Update the model tag to the beta tag for prototype 6.

Refs: #277

JessicaMeixner-NOAA commented:
Comparing with @shansun6 original input.nml (/scratch2/BMC/gsd-hpcs/Shan.Sun/FV3_RT/v16_35d_8cases_feb20/cpld_bmarkfrac_wave_v16_35d_2012010100_prod/input.nml) I believe the only missing difference is the &fv_core_nml namelist psm_bc = 1 

Yesterday, in the coupled modeling tag-up we also talked about setting the namelist &namsfc  FSICL to 1, but that could be a separate PR if need be (even along w/the psm_bc=1).

SMoorthi-emc commented:
Please set FSICL=0

On Thu, Mar 4, 2021 at 8:06 AM Jessica Meixner <notifications@github.com>
wrote:

> Comparing with @shansun6 <https://github.com/shansun6> original input.nml
> (/scratch2/BMC/gsd-hpcs/Shan.Sun/FV3_RT/v16_35d_8cases_feb20/cpld_bmarkfrac_wave_v16_35d_2012010100_prod/input.nml)
> I believe the only missing difference is the &fv_core_nml namelist psm_bc =
> 1
>
> Yesterday, in the coupled modeling tag-up we also talked about setting the
> namelist &namsfc FSICL to 1, but that could be a separate PR if need be
> (even along w/the psm_bc=1).
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/global-workflow/pull/281#issuecomment-790604343>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYQTGC6GBJSAFXPFCBLTB6AVXANCNFSM4YSJUXSQ>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: Shrinivas.Moorthi@noaa.gov
Phone:  (301) 683-3718   Fax: (301) 683-3718

JessicaMeixner-NOAA commented:
Thanks for that correction @SMoorthi-emc

WalterKolczynski-NOAA commented:
Testing the additions now. I wasn't sure about psm_bc since it wasn't being specified in the workflow at all. Now it is.

WalterKolczynski-NOAA committed:
Add additional settings for prototype 6

Two additional settings needed to be added for prototype 6:
psm_bc = 1
FSICL = 0

Refs: #277

WalterKolczynski-NOAA commented:
New sample namelist: https://gist.github.com/WalterKolczynski-NOAA/7aabc8908edd50f1d3d0dbe2980438a6

SMoorthi-emc commented:
Well, "ddmp" is still 0.1; may want to change to 0.2

On Fri, Mar 5, 2021 at 9:08 AM Walter Kolczynski - NOAA <
notifications@github.com> wrote:

> New sample namelist:
> https://gist.github.com/WalterKolczynski-NOAA/7aabc8908edd50f1d3d0dbe2980438a6
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/global-workflow/pull/281#issuecomment-791441133>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYSMLIUD7TSRFFQN3SLTCDQTXANCNFSM4YSJUXSQ>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: Shrinivas.Moorthi@noaa.gov
Phone:  (301) 683-3718   Fax: (301) 683-3718

JessicaMeixner-NOAA commented:
Yes, and I will fix that in a follow up PR.  Since that wasn't in the input.nml Walter was trying to match, I think it's an unfair expectation that he would have that fixed in this PR.

SMoorthi-emc commented:
Great.

WalterKolczynski-NOAA committed:
Adjust damping parameter for prototype 6

Adjusts the value of dddmp to 0.2 for prototype 6.

Refs: #277

WalterKolczynski-NOAA commented:
NB: I had to force-push because I amended the message in the previous commit because I referenced the PR number instead of the issue number.
